rapids-cmake/cpm/cccl.cmake

```diff
@@ -159,10 +159,14 @@ function(rapids_cpm_cccl)
   if(TARGET CCCL::CCCL)
     # Can be removed once we move to CCCL 2.3
```

Comment:
This comment seems outdated. @trxcllnt Should it be removed?

Reply:
I don't know

Reply:
The comment is old and should be removed, the code not so much